pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -98,9 +96,8 @@ def __init__(self,
         self._control_sock_wfile = self._control_sock.makefile('w')
 
         # if we're in a coroutine - set socket to non-blocking
```

Comment:
improve the comment
```suggestion
        # set socket to non-blocking so we are able to cancel the wait if needed
```

Reply:
I added this info plus a bit more detail, please check again.